src/Misc/Hooks.BugFixes.cpp

```diff
@@ -1136,6 +1136,27 @@ DEFINE_HOOK(0x4C75DA, EventClass_RespondToEvent_Stop, 0x6)
 
 #pragma endregion
 
+#pragma region TeamCloseRangeFix
+
+int __fastcall Check2DDistanceInsteadOf3D(AbstractClass* pSource, void* _, AbstractClass* pTarget)
+{
+	const auto sourceCoords = pSource->GetCoords();
+	const auto targetCoords = pTarget->GetCoords();
+
+	// Aircraft has its own unique treatment, and it will not be changed here
+	int distance = Game::F2I((pSource->IsInAir() && pSource->WhatAmI() != AbstractType::Aircraft) ? // Jumpjets or sth in the air ?
+		(Point2D { sourceCoords.X - targetCoords.X, sourceCoords.Y - targetCoords.Y }.Magnitude() * 2) : // bonus to units in the air
+		sourceCoords.DistanceFrom(targetCoords)); // Original
```

Comment:
```suggestion
	int distance = Game::F2I((pSource->IsInAir() && pSource->WhatAmI() != AbstractType::Aircraft) // Jumpjets or sth in the air ?
		? (Point2D { sourceCoords.X - targetCoords.X, sourceCoords.Y - targetCoords.Y }.Magnitude() * 2) // bonus to units in the air
		: sourceCoords.DistanceFrom(targetCoords)); // Original
```

The reason for this change is because we read code from left to right, so it's far easier to see the operator(s) at the beginning of the line than on the end.

Reply:
Get it.